### DISCUSSION TITLE
how to comment out a number of marked lines?

### DISCUSSION THREAD

**Discussion by AB-Stromer** [eclipse-theia/theia#13910] [Q&A]
I am new with Theia IDE, so please be patient with my very basic question. Different to my Pycharm and VSCode I cannot find here how to comment out a number of marked code lines (this simple #)
Thanks for any advice

**Comment by msujew** [comment]
Hey @AB-Stromer,

we use the same keybindings as VS Code to comment out code: Ctrl+K followed by Ctrl+C.

Note that this only works for languages that have a language configuration registered, as the editor doesn't know otherwise how a comment for that specific language looks like. Usually, any VS Code extension that contributes syntax highlighting also contributes a language configuration. Therefore, as long as your language is syntax highlighted, commenting out using the shortcuts mentioned above should work as expected.
